ShapePopulationViewer: Windows build error related to ShapePopulationBase.cxx

```
       "D:\D\S\SSALT-0-build\ALL_BUILD.vcxproj" (default target) (1) ->
       "D:\D\S\SSALT-0-build\ShapePopulationViewer.vcxproj" (default target) (27) ->
       (CustomBuild target) ->
         D:\D\S\SSALT-0-build\ShapePopulationViewer\src\ShapePopulationBase.cxx(503): error C2039: 'back_inser
ter': is not a member of 'std' [D:\D\S\SSALT-0-build\ShapePopulationViewer-build\src\ShapePopulationViewerLib.
vcxproj] [D:\D\S\SSALT-0-build\ShapePopulationViewer.vcxproj]
         D:\D\S\SSALT-0-build\ShapePopulationViewer\src\ShapePopulationBase.cxx(503): error C3861: 'back_inser
ter': identifier not found [D:\D\S\SSALT-0-build\ShapePopulationViewer-build\src\ShapePopulationViewerLib.vcxp
roj] [D:\D\S\SSALT-0-build\ShapePopulationViewer.vcxproj]
         D:\D\S\SSALT-0-build\ShapePopulationViewer\src\ShapePopulationBase.cxx(503): error C2039: 'back_inser
ter': is not a member of 'std' [D:\D\S\SSALT-0-build\ShapePopulationViewer-build\src\ShapePopulationViewerLib.
vcxproj] [D:\D\S\SSALT-0-build\ShapePopulationViewer.vcxproj]
         D:\D\S\SSALT-0-build\ShapePopulationViewer\src\ShapePopulationBase.cxx(503): error C3861: 'back_inser
ter': identifier not found [D:\D\S\SSALT-0-build\ShapePopulationViewer-build\src\ShapePopulationViewerLib.vcxp
roj] [D:\D\S\SSALT-0-build\ShapePopulationViewer.vcxproj]

    68 Warning(s)
    4 Error(s)
```

--- a/SuperBuild/External_ShapePopulationViewer.cmake
+++ b/SuperBuild/External_ShapePopulationViewer.cmake
@@ -54,7 +54,7 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
   ExternalProject_Add(${proj}
     ${${proj}_EP_ARGS}
     GIT_REPOSITORY "${EP_GIT_PROTOCOL}://github.com/jcfr/ShapePopulationViewer.git"
-    GIT_TAG "724374ca71ce58ec12bbe179debee93fb97ec118" # fix-configure-error-visual-studio
+    GIT_TAG "d8d75f163fc6b2650b9d034012eb559e948bd32b" # fix-configure-error-visual-studio
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${${proj}_DIR}
     INSTALL_COMMAND ${CMAKE_COMMAND} --build ${${proj}_PACKAGE_DIR} --config ${config} --target package

